hack/e2e.go

```diff
@@ -181,7 +181,7 @@ func Up() bool {
 
 // Ensure that the cluster is large engough to run the e2e tests.
 func ValidateClusterSize() {
-	// Check that there are at least 3 minions running
+	// Check that there are at least 2 minions running
```

Comment:
Can you please change 2 to "minMinionCount"?
If we decide to change that value in the future, this comment will again become obsolete.